### ISSUE TITLE
Combine SoccerPlayer.isActiveProperty into the SoccerPlayer.phaseProperty

### ISSUE THREAD

**Issue by samreid** [phetsims/center-and-variability#327]
Get rid of isActiveProperty.  Change poseProperty to phaseProperty. Discovered in https://github.com/phetsims/center-and-variability/issues/193

**Comment by samreid** [comment]
I'm not sure this is a win/win proposal.  For instance, in SoccerPlayerNode, it is nice that the poses all have a distinct associated image.  @matthew-blackman can you please inspect and advise?

**Comment by matthew-blackman** [comment]
@samreid I agree that the pose logic works well, but I still think that soccerPlayerNode can be improved by implementing a phaseProperty and moving the isActiveProperty logic into it.

For example, we could update the pose based on the phase, and also implement a screen-specific way to show/hide the players when they are done kicking in the different screens. So we'd have poseProperty and phaseProperty, but no isActiveProperty (or one that derives from phaseProperty and accounts for screen-specific logic).

Thoughts?

**Comment by samreid** [comment]
Good idea, let's do that, thanks.

**Comment by matthew-blackman** [comment]
The above commit seems to be working well aside from one bug that I'm tracking:

To address https://github.com/phetsims/center-and-variability/issues/328, in the Variability screen the soccer player should stay visible and standing after the last ball has been kicked. 

The above commit achieves this for all maxKicks except for maxKicks = 30. I'm thinking this may have something to do with maxKicksLimit but I haven't deeply investigated yet. @marlitas I'll mark this help-wanted so we can do a code review and look at this synchronously.

**Comment by matthew-blackman** [comment]
This is ready for review @marlitas. Let me know if you want to discuss anything synchronously.

**Comment by matthew-blackman** [comment]
That bug was addressed in a commit in https://github.com/phetsims/center-and-variability/issues/328.

**Comment by matthew-blackman** [comment]
Reviewed with @marlitas and all looks good. Closing.
